jobsapp/models.py

```diff
@@ -1,42 +1,94 @@
-    filled = models.BooleanField(default=False)
-    salary = models.IntegerField(default=0, blank=True)
+        (JOB_TYPE_FULL_TIME, _("Full time")),
+        (JOB_TYPE_PART_TIME, _("Part time")),
+        (JOB_TYPE_INTERNSHIP, _("Internship")),
+    )
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        verbose_name=_("User"),
+        help_text=_("User who creates this job."),
+    )
+    title = models.CharField(
+        max_length=300, verbose_name=_("Title"), help_text=_("Short job title.")
+    )
+    description = models.TextField(
+        verbose_name=_("Description"), help_text=_("Long job description.")
+    )
+    location = models.CharField(
+        max_length=150, verbose_name=_("Location"), help_text=_("Location for this job position.")
+    )
+    type = models.CharField(
+        choices=JOB_TYPES, max_length=10, verbose_name=_("Type"), help_text=_("Job type.")
+    )
+    category = models.CharField(
+        max_length=100, verbose_name=_("Category"), help_text=_("Category clasification.")
+    )
+    last_date = models.DateTimeField(verbose_name=_("Last date"), help_text=_("Last date."))
+    company_name = models.CharField(
+        max_length=100, verbose_name=_("Company"), help_text=_("Job's Company name.")
+    )
+    company_description = models.CharField(
+        max_length=300,
+        verbose_name=_("Company description"),
+        help_text=_("Company description, activity,..."),
+    )
+    website = models.CharField(
+        max_length=100, default="", verbose_name=_("Website"), help_text=_("Company Website URL.")
+    )
+    created_at = models.DateTimeField(
+        default=timezone.now, verbose_name=_("Created"), help_text=_("Job creation date and time.")
+    )
+    filled = models.BooleanField(
+        default=False, verbose_name=_("Filled"), help_text=_("Job position is filled.")
+    )
+    salary = models.IntegerField(
+        default=0, blank=True, verbose_name=_("Salary"), help_text=_("Maximum salary for this job.")
+    )
+
+    class Meta:
+        verbose_name = _("Job")
+        verbose_name_plural = _("Jobs")
 
     def __str__(self):
         return self.title
 
 
 class Applicant(models.Model):
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
-    job = models.ForeignKey(Job, on_delete=models.CASCADE, related_name="applicants")
-    created_at = models.DateTimeField(default=timezone.now)
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, verbose_name=_("User"), help_text=_("User applicant.")
+    )
+    job = models.ForeignKey(
+        Job,
+        on_delete=models.CASCADE,
+        related_name="applicants",
+        verbose_name=_("Job"),
+        help_text=_("Job in applicant."),
+    )
+    created_at = models.DateTimeField(
+        default=timezone.now,
+        verbose_name=_("Created"),
+        help_text=_("Applicant creation date and time."),
+    )
 
     class Meta:
+        verbose_name = _("Applicant")
```

Comment:
You are adding translations with ugettext everywhere, completely fine. But should you create the po files too? If you call `python manage.py makemessages` some po files should appear with these keys ready to be translated.
There is the #88 issue that is related to this. Maybe we can wait until we will have a decission on what to do with the translations.
Thanks Xavi, nice job :)

Reply:
Ok, now I read the other comments... XDD Fine. Then don't take this comment serious :)

Reply:
Hahahahahah

Reply:
lol